### PR TITLE
Add lang block to readme

### PR DESCRIPTION
**Pull request by Arhell**
issue #20360
/language uk

### CODE REVIEW

File: README-uk.md [hunk 1]
```diff
@@ -19,6 +19,19 @@
 * [Керівництво зі стилю оформлення документації](http://kubernetes.io/docs/contribute/style/style-guide/)
 * [Переклад документації Kubernetes іншими мовами](https://kubernetes.io/docs/contribute/localization/)
 
+## Файл `README.md` на інших мовах
```
Comment (butuzov) [inline README-uk.md:22]:
```suggestion
## Файл `README.md` іншими мовами
```

Comment (Arhell) [reply README-uk.md]: done, thanks!

**Comment by butuzov** [comment]
Hi Raja, can you approve changes to Ukrainian `README` file?
/assign @rajakavitha1

**Comment by sftim** [comment]
/approve
echoing https://github.com/kubernetes/website/pull/20380#issuecomment-614497866

**Pull request merged**